docs/source/markdown/podman-systemd.unit.5.md

```diff
@@ -481,15 +481,16 @@ There is only one required key, `Yaml`, which defines the path to the Kubernetes
 
 Valid options for `[Kube]` are listed below:
 
-| **[Kube] options**                | **podman kube play equivalent**        |
-| -----------------                 | ------------------                     |
-| ConfigMap=/tmp/config.map         | --config-map /tmp/config.map           |
-| LogDriver=journald                | --log-driver journald                  |
-| Network=host                      | --net host                             |
-| PodmanArgs=--annotation=key=value | --annotation=key=value                 |
-| PublishPort=59-60                 | --publish=59-60                        |
-| UserNS=keep-id:uid=200,gid=210    | --userns keep-id:uid=200,gid=210       |
-| Yaml=/tmp/kube.yaml               | podman kube play /tmp/kube.yaml        |
+| **[Kube] options**                | **podman kube play equivalent**             |
+| --------------------------------- | ------------------------------------------- |
+| ConfigMap=/tmp/config.map         | --config-map /tmp/config.map                |
+| ExitPolicy=any                    | --annotation=io.podman.kube.exit.policy=any |
```

Comment:
Not for this PR, just something to consider, it would be nice from a user standpoint if we could shorten this and drop `io.podman.kube`, or at the very least drop `io.podman`.  I don't know if that's feasible or not.  But something like:

```suggestion
| ExitPolicy=any                    | --annotation=kube.exit.policy=any |
```

Again, not for this PR, just something to ruminate upon.

Reply:
Actually, according at the [K8S specification for annotation](https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/#syntax-and-character-set), the key name may be divided into a prefix, an FQDN denoting the name of the service that will use this annotation, and name, representing the specific key's name. So, I would change it to `io.podman/kube-exit-policy`

Reply:
There is large number of other annotations using the format of `io.podman.foo.bar`.  I figured to be consistent.  I'll check what to do with annotation vs. CLI flag etc.